x/ccv/consumer/keeper/validators_test.go

```diff
@@ -93,60 +97,79 @@ func (k KeeperTestSuite) TestApplyCCValidatorChanges() {
 			changes: []abci.ValidatorUpdate{
 				{PubKey: changes[0].PubKey, Power: changes[0].Power + 1},
 				{PubKey: changes[1].PubKey, Power: changes[1].Power + 2},
-				{PubKey: changes[2].PubKey, Power: changes[1].Power + 3},
-				{PubKey: changes[3].PubKey, Power: changes[1].Power + 4},
+				{PubKey: changes[2].PubKey, Power: changes[2].Power + 3},
```

Comment:
Here's another change in testing logic. Previously the IBC testing suite had all validator powers start equal. So ```changes[1].Power``` vs ```changes[2].Power``` didn't matter. 

In the new test, 4 validators start out with powers of 1,2,3, and 4 respectively. So indexing here now matters